dirsrvtests/tests/suites/indexes/regression_test.py

```diff
@@ -12,14 +12,74 @@
 import ldap
 from lib389._constants import DEFAULT_BENAME, DEFAULT_SUFFIX
 from lib389.index import Indexes
-from lib389.backend import Backends
+from lib389.backend import Backends, DatabaseConfig
 from lib389.idm.user import UserAccounts
+from lib389.idm.group import Groups, Group
 from lib389.topologies import topology_st as topo
 from lib389.utils import ds_is_older
+from lib389.plugins import MemberOfPlugin
 
 pytestmark = pytest.mark.tier1
 
 
+@pytest.fixture(scope="function")
+def add_a_group_with_users(request, topo):
+    """
+    Add a group and users, which are members of this group.
+    """
+    groups = Groups(topo.standalone, DEFAULT_SUFFIX, rdn=None)
+    group = groups.create(properties={'cn': 'test_group'})
+    users_list = []
+    users_num = 100
+    users = UserAccounts(topo.standalone, DEFAULT_SUFFIX, rdn=None)
+    for num in range(users_num):
+        USER_NAME = f'test_{num}'
+        user = users.create(properties={
+            'uid': USER_NAME,
+            'sn': USER_NAME,
+            'cn': USER_NAME,
+            'uidNumber': f'{num}',
+            'gidNumber': f'{num}',
+            'homeDirectory': f'/home/{USER_NAME}'
+        })
+        users_list.append(user)
+        group.add_member(user.dn)
+
+    def fin():
+        """
+        Removes group and users.
+        """
+        # If the server crashed, start it again to do the cleanup
+        if not topo.standalone.status():
+            topo.standalone.start()
+        for user in users_list:
+            user.delete()
+        group.delete()
+
+    request.addfinalizer(fin)
+
+
+@pytest.fixture(scope="function")
+def set_small_idlistscanlimit(request, topo):
+    """
+    Set nsslapd-idlistscanlimit to a smaller value to accelerate the reproducer
+    """
+    db_cfg = DatabaseConfig(topo.standalone)
+    db_cfg.set([('nsslapd-idlistscanlimit', '50')])
```

Comment:
According to the [documentation](https://access.redhat.com/documentation/en-us/red_hat_directory_server/11/html/configuration_command_and_file_reference/database_plug_in_attributes#nsslapd_idlistscanlimit), the min value of 'nsslapd-idlistscanlimit' is 100.

Reply:
Nice catch! Fixed.